pkg/connector/client.go

```diff
@@ -373,3 +374,16 @@ func (wa *WhatsAppClient) syncRemoteProfile(ctx context.Context, ghost *bridgev2
 	}
 	zerolog.Ctx(ctx).Info().Msg("Remote profile updated")
 }
+
+func (wa *WhatsAppClient) HandleMatrixViewingChat(ctx context.Context, msg *bridgev2.MatrixViewingChat) error {
+	// Reset portal last sync time so we immediately sync (next time the queue runs)
+	msg.Portal.Metadata.(*waid.PortalMetadata).LastSync.Time = time.Time{}
```

Comment:
Immediate reset feels a little aggressive, might be worth checking if the last sync was <5 minutes ago and not running if so.

Reply:
https://github.com/mautrix/whatsapp/pull/797/commits/a1e7be6428323ba2742d525579a41cc03474bbd9